EOLib.Graphics/NativeGraphicsLoader.cs

```diff
@@ -23,14 +23,14 @@ public IImage LoadGFX(GFXTypes file, int resourceValue)
             }
             catch (ArgumentException)
             {
-#if DEBUG
+#if DEBUG && THROW_ON_TEXTURE_FAILURE
```

Comment:
I _want_ this to throw by default if a texture is missing.
a) Main clone is currently using fanfic textures. If one is missing, updating those is an exercise for the user. I'll support them eventually but not yet
b) Connecting to a server with custom assets (e.g. BU) is also an exercise for the user

I don't want potential gfx loading issues to be hidden. In release mode it's fine to not crash, in debug mode it should be obvious that something is wrong

Reply:
Ah apologies I thought I removed this from the pr, I checked it in by mistake. It was so that I could experience bald missing graphics on clone without crashing 💀